apps/design/frontend/src/document_svg.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Box, Color, Ellipse, Page, Rectangle } from './document_types';
+
+type SvgRectangleProps = Omit<Rectangle, 'type'>;
+
+export function SvgRectangle({
+  borderRadius,
+  ...props
+}: SvgRectangleProps): JSX.Element {
+  return <rect {...props} rx={borderRadius} ry={borderRadius} />;
+}
+
+type SvgEllipseProps = Omit<Ellipse, 'type'>;
+
+export function SvgEllipse({
+  x,
+  y,
+  width,
+  height,
+  ...props
+}: SvgEllipseProps): JSX.Element {
+  return (
+    <ellipse
+      cx={x + width / 2}
+      cy={y + height / 2}
+      rx={width / 2}
+      ry={height / 2}
+      {...props}
+    />
+  );
+}
+
+interface SvgBoxProps extends Omit<Box, 'type' | 'children'> {
+  children?: React.ReactNode;
+}
+
+export function SvgBox({
```

Comment:
If we were productionizing this, I might suggest a different name for `SvgBox` since it's hard to disambiguate from `SvgRectangle` at the moment

Reply:
Thanks, good feedback!